airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -180,6 +197,8 @@ def get_task_instances_batch(session=None):
         data = task_instance_batch_form.load(body)
     except ValidationError as err:
         raise BadRequest(detail=str(err.messages))
+    state = data['state']
+    states = _convert_state(state)
```

Comment:
```suggestion
    states = _convert_state(data['state'])
```

Let’s avoid the `state` local variable to avoid using the wrong name accidentally.